content/cpp/concepts/polymorphism/polymorphism.md

```diff
@@ -0,0 +1,115 @@
+---
+Title: 'Polymorphism'
+Description: 'The word Polymorphism means having many form. Polymorphism is an important concept of object-oriented programming. It simply means more than one form. That is, the same entity (function or operator) can behave differently in different scenarios.'
```

Comment:
```suggestion
Description: 'Polymorphism is an important concept of object-oriented programming. It simply means more than one form. That is, the same entity (function or operator) can behave differently in different scenarios.'
```

Reply:
made the requested changes successfully.